### ISSUE TITLE
Half of sprite of some guy is white

### ISSUE THREAD

**Issue by Aliceee2ch** [tgstation/tgstation#87583]
### Client Version:

515.1642

### Issue Summary:

Cant describe much, since ive seen only that guy had half of their sprite as white rectangle. They were in some MOD and probably upside down, rod of asclepius aswell.
![image](https://github.com/user-attachments/assets/74249fb0-c32d-4aae-9556-21de8f5bfc8b)

### Round ID:

241041

### Reproduction Steps:

1. N/A

**Comment by MrMelbert** [comment]
Really good guess but I think it's the one armed glove mask

**Comment by mc-oofert** [comment]
i have no clue why it does this honestly

**Comment by MrMelbert** [comment]
I assume it only happens in the alert popup and not in world, which just implies it's some jank with appearance copying

**Comment by Aliceee2ch** [comment]
> I assume it only happens in the alert popup and not in world, which just implies it's some jank with appearance copying

not exactly, i clearly remember this guy sprite was white ingame aswell, screenshot is just thing that I could've catch up.

**Comment by mc-oofert** [comment]
one thing worth mentioning is that while in the popup the character faces south the mask faces the actual direction somehow

**Comment by SmArtKar** [comment]
> one thing worth mentioning is that while in the popup the character faces south the mask faces the actual direction somehow, aside from that, cant reproduce also the popup doesnt seem to care if you actually gain the limb back (via VV atleast)

@mc-oofert looks like a sprite for glove masking for a missing right arm?
![image](https://github.com/user-attachments/assets/03e2c3d3-c146-4054-a67a-7e7ca0f6586e)

**Comment by mc-oofert** [comment]
> > one thing worth mentioning is that while in the popup the character faces south the mask faces the actual direction somehow, aside from that, cant reproduce also the popup doesnt seem to care if you actually gain the limb back (via VV atleast)
> 
>  -mention- looks like a sprite for glove masking for a missing right arm? -image-

yeah i knew what sprite it was no clue why it does it though
